internal/provider/provider.go

```diff
@@ -81,9 +81,23 @@ func New(version string) func() *schema.Provider {
 				"oauth_scopes": {
 					Description: "The list of the scopes required for your application (for a list of possible scopes, see " +
 						"[Authorize requests](https://developers.google.com/admin-sdk/directory/v1/guides/authorizing))",
-					Type:     schema.TypeList,
+					Type:     schema.TypeSet,
```

Comment:
Is there a reason this would be `TypeSet`? I think I'd prefer to keep it a list if possible, lists are much easier to work with.

Reply:
@megan07 I mainly based it on the docs description of Set being where you don't care about the order and List where you do. I think the former is more correct but TBH I don't know if it really matters for this type of schema anyways. I've reverted in my more recent commits.